app/views/user_mentee_applications/show.html.erb

```diff
@@ -1,70 +1,94 @@
-<div class="max-w-prose mx-auto p-4">
-  <section class="flex flex-col mb-3 mb-5 ">
-    <div class="mx-auto text-2xl text-red-500 mb-3">
-      <%= @user_mentee_application.current_status.to_s.humanize %>
+<div class="mx-auto p-4 my-8 max-w-prose">
+  <section class="flex flex-col my-4 mx-auto rounded-md border-2">
+    <div class="inline my-4 mx-auto pt-4">
+      <span class="text-md text-base-content/70">Application Status</span>
+      <span class="text-xl font-bold tracking-wide ml-8">
+        <%= @user_mentee_application.current_status.to_s.humanize %>
+      </span>
     </div>
-
-    <ul class="menu menu-horizontal bg-base-200">
-      <li> <%= render(UserMenteeApplication::ActionButtonBuilderComponent.new(mentee_application: @user_mentee_application)) %> </li>
-    </ul>
+    <span class="pb-4 inline mx-auto">
+      <% if @user_mentee_application.user == current_user %>
+        <p class="text-md p-6">
+          Thank you for submitting your application to join us at the <span class="font-semibold">Agency of Learning</span>!<br /><br />
```

Comment:
Need copy to communicate expectations and the status of the application to applicants.